classes/submission/Repository.php

```diff
@@ -961,6 +961,19 @@ protected function mapDashboardViews(Collection $types, Context $context, User $
                         'reviews',
                         ['reviewsSubmitted' => true]
                     );
+                case DashboardView::TYPE_REVIEWS_OVERDUE:
+                    $collector = Repo::submission()->getCollector()
+                        ->filterByContextIds([$context->getId()])
+                        ->filterByOverdue(true)
+                        ->filterByStatus([PKPSubmission::STATUS_QUEUED]);
+                    return new DashboardView(
```

Comment:
This one is quite intense :-). There is good chance I am not interpreting things correctly..  so my concerns might be off.

I wonder whether we should filter by review stage? Otherwise even if we leave some overdue review in review stage and move it to editorial it still would show up?

Similar concern about current review round - should not we checking just that?

Reply:
Oh these concerns are probably covered with this part, right?
```
                    ->whereNotIn('rr.status', [
                        ReviewRound::REVIEW_ROUND_STATUS_RESUBMIT_FOR_REVIEW,
                        ReviewRound::REVIEW_ROUND_STATUS_SENT_TO_EXTERNAL,
                        ReviewRound::REVIEW_ROUND_STATUS_ACCEPTED,
                        ReviewRound::REVIEW_ROUND_STATUS_DECLINED,
                    ])
```

Reply:
My initial change included a new filter, I can re-add that.